### DISCUSSION TITLE
Client Libraries as NPM Packages?

### DISCUSSION THREAD

**Discussion by aaronatbissell** [aws/aws-connected-device-framework#39] [Ideas]
Any thoughts on releasing the client libraries (assetlibrary-client, provisioning-client, etc) as node packages published to the public npm registry so they can be easily consumed/used by down-stream projects?

**Comment by boardthatpowder** [comment]
Short answer... yes. This is something on our list to set up, but no ETA.

**Reply by aaronatbissell** [reply]
Awesome - Thanks!
